packages/domain-picker/src/domain-picker/style.scss

```diff
@@ -232,8 +232,7 @@ $accent-blue: #117ac9;
 }
 
 .domain-picker__suggestion-item-name {
-	flex-grow: 1;
-	flex-basis: 2px;
+	width: max-content;
```

Comment:
> "For text content this means that the content will not wrap at all even if it causes overflows" - .

The `max-content` property will prevent content from wrapping and will eventually cause overflows (see [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/CSS/max-content)). In our case, this happens when the URL is (extremely) long (see screenshot). Do you think we need to address and fix this? 

<img width="923" alt="Screenshot 2021-02-10 at 15 27 11" src="https://user-images.githubusercontent.com/12104589/107523074-a74c1a80-6bb4-11eb-95a8-5cb3d3b047ee.png">